politeiawww/cmd/piwww/piwww.go

```diff
@@ -90,8 +72,24 @@ type piwww struct {
 	VoteSummaries   voteSummariesCmd   `command:"votesummaries"`
 	VoteInventory   voteInventoryCmd   `command:"voteinventory"`
 
-	// TODO rename to reflect that its a users route
-	ProposalPaywall proposalPaywallCmd `command:"proposalpaywall"`
+	// User commands
+	UserNew                userNewCmd                   `command:"usernew"`
```

Comment:
Don't move all of these commands.  Just add the new command to the existing list.

Reply:
ops another rebase mistake, mb. fixed this